### ISSUE TITLE
Portal commands break when you use /portal set_portal_nbt {unbreakable:1b}

### ISSUE THREAD

**Issue by wowthiscooliishappy** [qouteall/ImmersivePortalsModForForge#156]
Portal commands dont work after you use /portal set_portal_nbt {unbreakable:1b}

**Comment by qouteall** [comment]
Try to close the game and reopen the game, does the issue remain?

**Comment by wowthiscooliishappy** [comment]
No, because when I exit minecraft world doesnt save because it is stuck in an infinite saving screen, there for I have to use task manager to close it. Also, I forgot to mention this, but the portal commands seem to only break if you try to make an obsidian portal unbreakable. With portal created with the make portal command can be set unbreakable and not break the commands.

**Comment by VexianEmpire** [comment]
Portals and Nether Portals are actually separate entities. Maybe Nether Portals aren't compatible with the `{Unbreakable}` tag?

**Comment by PoolloverNathan** [comment]
It worked fine for me when I last used it.
